src/values/ObjectValue.js

```diff
@@ -267,6 +268,7 @@ export default class ObjectValue extends ConcreteValue {
   properties: Map<string, PropertyBinding>;
   symbols: Map<SymbolValue, PropertyBinding>;
   unknownProperty: void | PropertyBinding;
+  temporalAlias: void | AbstractObjectValue;
```

Comment:
Should this be tracked? What happens if snapshots are obtained along different execution paths?